src/Misc/WaveShapeSmps.cpp

```diff
@@ -79,16 +123,18 @@ void waveShapeSmps(int n,
             break;
         case 7:
             ws = powf(2.0f, -ws * ws * 8.0f); //Limiter
+            par = par/4;
```

Comment:
Is this change really wanted? It will make all previous savefiles sound different, i.e. we need to patch the savefiles.

E.g. a sine wave with limiter sounds way more quiet after your patch.

Reply:
par is a new one. it originates in my patch "WIP new distorsion types tanh, cubic, square with Ptype 14 15 and 16"
therefore it should not be present in savefiles.

Reply:
Oh, of course. Good point.